src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,57 @@
  */
 
 public class Plane {
+    private String name;
+    private String type;
+    private int maxSpeed;
+    private int maxSeats;
+    private double maxCargo;
 
+    private Plane(PlaneBuilder builder) {
+        this.name = builder.name;
+        this.type = builder.type;
+        this.maxSpeed = builder.maxSpeed;
+        this.maxCargo = builder.maxCargo;
+        this.maxSeats = builder.maxSeats;
+    }
+
+    public static class PlaneBuilder {
+        private String name;
+        private String type;
+        private int maxSpeed;
+        private double maxCargo;
+        private int maxSeats;
+
+        private PlaneBuilder() {
+        }
```

Comment:
try to create PlaneBuilder, set few fields and then create Plane out of it 
pass your code as answer for this comment( don't push it to git)

Reply:
PlaneBuilder pb = new PlaneBuilder();
        pb.setMaxCargo(10);
        pb.setMaxSeats(23);
        Plane newPlane = pb.build();

Reply:
originally there was public static factory method to create builder, I eliminate it to conform to other solutions. I can revert it.

Reply:
added public method builder() to access private constructor of PlaneBuilder and pushed it to git